### DISCUSSION TITLE
Disabling Welcome shorcuts screen

### DISCUSSION THREAD

**Discussion by Okanda** [YaLTeR/niri#442] [Q&A] [answered]
Hello. 

I don't find how to disable the shortcuts window launched at the start of Niri. 

I think it is in the config file but. . . ? 

Thx.

**Comment by YaLTeR** [comment, marked as answer]
Here's the flag: https://github.com/YaLTeR/niri/wiki/Configuration:-Miscellaneous#hotkey-overlay

**Comment by Okanda** [comment]
Thank you :)
